xtask/src/arch.rs

```diff
@@ -1,19 +1,18 @@
-use anyhow::{anyhow, Error, Result};
 use std::fmt;
-use std::str::FromStr;
 
-#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+#[derive(Clone, Copy, Debug, Eq, PartialEq, clap::ValueEnum)]
 pub enum UefiArch {
+    #[value(name = "aarch64")]
```

Comment:
Isn't there a possibility to use "snake_case" on the strict level and omit the attributes for the variants?

Reply:
It doesn't quite work right for this case: `a_arch64` instead of `aarch64`.

Reply:
ah I see